macros/ModuleRunner.py

```diff
@@ -14,7 +14,7 @@ def __init__(self):
 
 class ModuleRunner(GenericPath, Constants):
     ''' Class container for list of objects for particular year '''
-    def __init__(self, module, years, runs, campaigns, jecs=None, extraName=None, extra_info={}):
+    def __init__(self, module, years, runs, campaigns, jecs=None, extraName=None, extra_info={}, jec_level = "default"):
```

Comment:
is default the right one? Not sure how it's used within bamboo

Reply:
In Bamboo it is def configureJets(
        variProxy, jetType, jec=None, jecLevels="default", smear=None,...)
So "default" would be the default in Bamboo. 

https://gitlab.cern.ch/cms-analysis/general/bamboo/-/blob/master/bamboo/analysisutils.py#L892